src/io/detail/GeantPhysicsList.hh

```diff
@@ -3,25 +3,25 @@
 // See the top-level COPYRIGHT file for details.
 // SPDX-License-Identifier: (Apache-2.0 OR MIT)
 //---------------------------------------------------------------------------//
-//! \file PhysicsList.hh
+//! \file GeantPhysicsList.hh
 //---------------------------------------------------------------------------//
 #pragma once
 
 #include <G4VUserPhysicsList.hh>
 
-namespace geant_exporter
+namespace celeritas
+{
+namespace detail
 {
 //---------------------------------------------------------------------------//
 /*!
  * Construct a user-defined physics list of particles and physics processes.
  */
-class PhysicsList : public G4VUserPhysicsList
+class GeantPhysicsList : public G4VUserPhysicsList
 {
   public:
-    // Construct empty
-    PhysicsList();
-    // Default destructor
-    ~PhysicsList();
+    // Set up during construction
+    GeantPhysicsList();
 
     // Set up minimal E.M. particle list
```

Comment:
I see that you've updated the comment in the .cc file to use `EM` instead of `E.M.`. So I guess we should update it on the header for consistency.

Reply:
It was mostly because Doxygen stops reading the "brief" header at `. ` -- so it affects documentation but doesn't really matter in headers. However I think `E.M.` is actually incorrect since the phrase is either hyphenated (old style) or a single word (electromagnetic).